app/assets/javascripts/tophat_space.coffee

```diff
@@ -0,0 +1,5 @@
+$('document').ready ->
+  # checks if tophat exists
+  tophatExist = $(".th-wrapper")
+  if tophatExist.length
```

Comment:
I was going to say you should add a ` > 0` here, but upon further research it looks like this is a widely accepted way to do this. However, I will suggest one final change, you can shorten this by making it one line: `if $('.th-wrapper).length`